src/teaching-element/Assessment/MultipleChoice.vue

```diff
@@ -39,9 +39,11 @@ export default {
     return { userAnswer: this.submission || [] };
   },
   computed: {
+    MCOptions() {
+      return { ...defaults, ...this.options.multipleChoice };
+    },
     type() {
```

Comment:
This computed is not needed anymore since now you know that `config` has `type` prop so you can use it directly (`this.config.type`) in the method below.

Reply:
True that although keeping it as is isn't that much of a sin :grin: